ext/hal/st/stm32cube/stm32f3xx/README

```diff
@@ -5,6 +5,7 @@ Patch List:
 *Changes from official delivery:
 -dos2unix applied
 -trailing white spaces removed
+-update LSI_VALUE to 40 KHz in file drivers/include/stm32f3xx_ll_rcc.h
```

Comment:
Could you make it as a new *paragraph with:
-Impacted file:
-ST Bug tracker ID: 37418

Reply:
done